rslidar_pointcloud/src/rawdata.cc

```diff
@@ -861,20 +872,32 @@ void RawData::unpack(const rslidar_msgs::rslidarPacket& pkt, pcl::PointCloud<pcl
         float arg_horiz = (float)azimuth_corrected / 18000.0f * M_PI;
         float arg_horiz_orginal = arg_horiz;
 
-        if (start_clip_angle_< end_clip_angle_)
+        bool skip = false;
+        for (const auto& cliping : clip_angles_)
```

Comment:
Typo in variable name.

Also, why don't you use structured bindings, if you change to C++17 anyway?